engine/topology/src/main/java/org/opennms/oce/engine/topology/shell/GenerateGraphML.java

```diff
@@ -61,7 +62,10 @@
 
     @Override
     public Object execute() throws IOException {
-        final Model model = ModelBuilderImpl.buildModel(inventoryDatasource.getInventory());
```

Comment:
We replace this one line with 4 lines in multiple places. Can we make a similar utility function that accepts the inventory and returns the model instead?